mono/metadata/threadpool-worker-default.c

```diff
@@ -221,6 +221,7 @@ rand_next (gpointer *handle, guint32 min, guint32 max)
 static void
 destroy (gpointer data)
 {
+#if 0
```

Comment:
This is opaque. If this is a known decision to allow a leak in order to prevent a crash, please add a comment like "This will leak every time [X] occurs, this is a quick fix to prevent [Y crash]".